sidebars.ts

```diff
@@ -56,6 +56,7 @@ const sidebars: SidebarsConfig = {
             'user-documentation/moderne-platform/how-to-guides/accessing-the-moderne-api',
             'user-documentation/moderne-platform/how-to-guides/recipe-execution-and-commits-with-graphql',
             'user-documentation/moderne-platform/how-to-guides/find-pii',
+            'user-documentation/moderne-platform/how-to-guides/custom-dev-center',
```

Comment:
Fyi @Laurens-W - in order for new pages to appear in the sidebar, you need to upgrade this file and include a link to the file. I've gone ahead and done that but figured I'd call it out so you know for the future.